### PR TITLE
[TEST] update org mapping

### PR DESCRIPTION
**Pull request by bumblebee89**

### CODE REVIEW

File: map_org_names.yaml [hunk 1]
```diff
@@ -211,6 +213,7 @@ mappings:
   - ['^[[:space:]]*align[[:space:]]technology[[:space:]]*$','Align Technology, Inc.']
   - ['^[[:space:]]*alion[[:space:]]science[[:space:]]and[[:space:]]technology[[:space:]]*$','Alion Science and Technology Corporation']
   - ['^[[:space:]]*(alipay\\.?com)[[:space:]]*$','Alipay']
+  - ['^[[:space:]]*aliste[[:space:]]*$',"ALISTE ConstrucciÃ³ i gestiÃ³ d'actius Inmobiliaris SL"]
```
Comment (lukaszgryglicki) [inline map_org_names.yaml:216]: Is this `Ã³` expected?

File: map_org_names.yaml [hunk 2]
```diff
@@ -137,6 +138,7 @@ mappings:
   - ['^[[:space:]]*ae[[:space:]]nv/sa[[:space:]]*$','AE - architects for business & ICT']
   - ['^[[:space:]]*aegon[[:space:]]*$','Aegon N.V.']
   - ['^[[:space:]]*amiedu[[:space:]]*$','AEL-Amiedu Oy']
+  - ['^[[:space:]]*aerokube[[:space:]]*$','Aerokube Software OÃœ']
```
Comment (lukaszgryglicki) [inline map_org_names.yaml:141]: `Ãœ` - is this expected?

File: map_org_names.yaml [hunk 3]
```diff
@@ -305,6 +308,7 @@ mappings:
   - ['^[[:space:]]*animal[[:space:]]logic[[:space:]]*$','Animal Logic Pty Ltd']
   - ['^[[:space:]]*animo[[:space:]]solutions[[:space:]]*$','animosolution']
   - ['^[[:space:]]*anon[[:space:]]solutions[[:space:]]inc\\.?[[:space:]]*$','Anon Solutions']
+  - ['^[[:space:]]*akdb[[:space:]]*$','Anstalt fÃ¼r Kommunale Datenverarbeitung in Bayern (AKDB)']
```
Comment (lukaszgryglicki) [inline map_org_names.yaml:311]: `fÃ¼r`?

File: map_org_names.yaml [hunk 4]
```diff
@@ -420,7 +425,7 @@ mappings:
   - ['^[[:space:]]*asipem[[:space:]]*$','ASIPEM']
   - ['^[[:space:]]*ask[[:space:]]*$','Ask Media Group, LLC']
   - ['^[[:space:]]*askbot[[:space:]]*$','askbot7']
-  - ['^[[:space:]]*askul[[:space:]]co\\.?,?[[:space:]]l\\td.?[[:space:]]*$','ASKUL Corporation']
+  - ['^[[:space:]]*askul[[:space:]]co\\.?,?[[:space:]]ltd\\.?[[:space:]]*$','ASKUL Corporation']
```
Comment (lukaszgryglicki) [inline map_org_names.yaml:428]: Good catch! :D

File: map_org_names.yaml [hunk 5]
```diff
@@ -516,8 +521,10 @@ mappings:
   - ['^[[:space:]]*axivion[[:space:]]*$','Axivion GmbH']
   - ['^[[:space:]]*axmor[[:space:]]*$','Axmor Software']
   - ['^[[:space:]]*axoni[[:space:]]*$','Axonista']
+  - ['^[[:space:]]*axway[[:space:]]appcelerator[[:space:]]*$','Axway Software SA']
   - ['^[[:space:]]*ayanworks[[:space:]]*$','AyanWorks Technology Solutions Pvt. Ltd.']
   - ['^[[:space:]]*retailio[[:space:]]*$','Ayro Retail Solutions Private Limited']
+  - ['^[[:space:]]*az[[:space:]]informática[[:space:]]*$','AZ Tecnologia em GestÃ£o']
```
Comment (lukaszgryglicki) [inline map_org_names.yaml:527]: `Ã£` ?

File: map_org_names.yaml [hunk 6]
```diff
@@ -626,6 +634,7 @@ mappings:
   - ['^[[:space:]]*berkeleytime[[:space:]]*$','Berkeleytime, LLC']
   - ['^[[:space:]]*(bernie[[:space:]]a\\.?i\\.?)[[:space:]]*$','Bernie']
   - ['^[[:space:]]*bestbytes[[:space:]]*$','bestbytes GmbH']
+  - ['^[[:space:]]*beta[[:space:]]educação[[:space:]]*$','Beta EducaÃ§Ã£o Ltda Me']
```
Comment (lukaszgryglicki) [inline map_org_names.yaml:637]: `Ã§Ã£` - please look for other such (encoding?) issues...

**Review by lukaszgryglicki — CHANGES_REQUESTED** [review]
Encoding issues, see my comments - there are more probably.

File: map_org_names.yaml (hunk 1 above)
Comment (bumblebee89) [reply map_org_names.yaml:216]: @lukaszgryglicki yes, the names (with encoding) are how SFDC has it. I get the names by querying the org service

Comment (lukaszgryglicki) [reply map_org_names.yaml:216]: Yes but this looks awful, maybe the service returns then wrong - I would rather convert to a correct name with a correct encoding (for example google for that name).
Example (ggogle): `ALISTE ConstrucciÃ³ i gestiÃ³ d'actius Inmobiliaris` -> `A²/Aliste-Construccions i Gestió d'Actius Immobiliaris`
Also what @1010sachin & @fayazg think about this?

Comment (bumblebee89) [reply map_org_names.yaml:216]: @lukaszgryglicki yeah, i agree but AFAIK, the affiliations service depends on our org names being exactly the same as what is in their database hence why I use their values as my source of truth

Comment (lukaszgryglicki) [reply map_org_names.yaml:216]: OK then, I'm changing to approved.

**Review by lukaszgryglicki — APPROVED** [review]
/lgtm

**Comment by bumblebee89** [comment]
@lukaszgryglicki merged. please run org_map endpoint when you have sometime. Thanks

**Comment by lukaszgryglicki** [comment]
PR should be for `test` not `master` - I've run after the Pr was merged and there were no updates. I ported this PR's changes to `test` run again and:
```
{"text":"Updated organization '1&1 Internet AG' -> '1&1 Telecommunication SE' on 9 enrollments, Updated organization '1&1 Internet AG' -> '1&1 Telecommunication SE' on 24 archived enrollments, Deleted organization '1&1 Internet AG' (id=30112), Updated organization name '4linux' -> '4Linux' (id=52992), Updated organization 'Accenture plc' -> 'Accenture Global Solutions Limited' on 30 enrollments, Updated organization 'Accenture plc' -> 'Accenture Global Solutions Limited' on 24 archived enrollments, Deleted organization 'Accenture plc' (id=58995), Updated organization 'Schibsted Spain' -> 'Adevinta ASA' on 8 enrollments, Updated organization 'Schibsted Spain' -> 'Adevinta ASA' on 5 archived enrollments, Deleted organization 'Schibsted Spain' (id=44931), Added organization 'Aerokube Software OÃœ' (id=77300), Updated organization 'Aerokube' -> 'Aerokube Software OÃœ' on 8 enrollments, Deleted organization 'Aerokube' (id=43294), Added organization 'ALISTE ConstrucciÃ³ i gestiÃ³ d'actius Inmobiliaris SL' (id=77301), Updated organization 'Aliste' -> 'ALISTE ConstrucciÃ³ i gestiÃ³ d'actius Inmobiliaris SL' on 5 enrollments, Updated organization 'Aliste' -> 'ALISTE ConstrucciÃ³ i gestiÃ³ d'actius Inmobiliaris SL' on 6 archived enrollments, Deleted organization 'Aliste' (id=42244), Added organization 'Anstalt fÃ¼r Kommunale Datenverarbeitung in Bayern (AKDB)' (id=77302), Updated organization 'AKDB' -> 'Anstalt fÃ¼r Kommunale Datenverarbeitung in Bayern (AKDB)' on 12 enrollments, Deleted organization 'AKDB' (id=46587), Added organization 'AquaQ Analytics Limited' (id=77303), Updated organization 'AquaQ Analytics' -> 'AquaQ Analytics Limited' on 7 enrollments, Updated organization 'AquaQ Analytics' -> 'AquaQ Analytics Limited' on 7 archived enrollments, Deleted organization 'AquaQ Analytics' (id=52906), Updated organization 'DDP Asia' -> 'Ardis Technologies BV' on 5 enrollments, Updated organization 'DDP Asia' -> 'Ardis Technologies BV' on 5 archived enrollments, Deleted organization 'DDP Asia' (id=51254), Updated organization 'ARM' -> 'Arm Limited' on 1313 enrollments, Updated organization 'ARM' -> 'Arm Limited' on 2926 archived enrollments, Deleted organization 'ARM' (id=26216), Updated organization 'ASKUL Co., Ltd.' -> 'ASKUL Corporation' on 5 enrollments, Updated organization 'ASKUL Co., Ltd.' -> 'ASKUL Corporation' on 7 archived enrollments, Deleted organization 'ASKUL Co., Ltd.' (id=54463), Updated organization 'Automatic Data Processing, Inc.' -> 'Automatic Data Processing, Inc. (ADP)' on 105 enrollments, Updated organization 'Automatic Data Processing, Inc.' -> 'Automatic Data Processing, Inc. (ADP)' on 461 archived enrollments, Deleted organization 'Automatic Data Processing, Inc.' (id=56660), Added organization 'Axway Software SA' (id=77304), Updated organization 'Axway Appcelerator' -> 'Axway Software SA' on 8 enrollments, Deleted organization 'Axway Appcelerator' (id=47080), Added organization 'AZ Tecnologia em GestÃ£o' (id=77305), Updated organization 'AZ Informática' -> 'AZ Tecnologia em GestÃ£o' on 7 enrollments, Updated organization 'AZ Informática' -> 'AZ Tecnologia em GestÃ£o' on 16 archived enrollments, Deleted organization 'AZ Informática' (id=46072), Updated organization 'Detica' -> 'BAE Systems' on 11 enrollments, Deleted organization 'Detica' (id=31490), Updated organization 'Battelle' -> 'Battelle Memorial Institute' on 6 enrollments, Updated organization 'Battelle' -> 'Battelle Memorial Institute' on 6 archived enrollments, Deleted organization 'Battelle' (id=49483), Added organization 'Beta EducaÃ§Ã£o Ltda Me' (id=77306), Updated organization 'Beta Educação' -> 'Beta EducaÃ§Ã£o Ltda Me' on 6 enrollments, Deleted organization 'Beta Educação' (id=42364), Updated organization 'Bloomberg LP' -> 'Bloomberg L.P.' on 5 enrollments, Updated organization 'Bloomberg LP' -> 'Bloomberg L.P.' on 7 archived enrollments, Deleted organization 'Bloomberg LP' (id=58660), Added organization 'BOA VISTA SERVIÃ‡OS SA' (id=77307), Updated organization 'Boa Vista SCPC' -> 'BOA VISTA SERVIÃ‡OS SA' on 19 enrollments, Deleted organization 'Boa Vista SCPC' (id=45963), Updated organization 'Beijing ByteDance Network Technology Co., Ltd.' -> 'Bytedance Ltd' on 106 enrollments, Updated organization 'Beijing ByteDance Network Technology Co., Ltd.' -> 'Bytedance Ltd' on 236 archived enrollments, Deleted organization 'Beijing ByteDance Network Technology Co., Ltd.' (id=47370), Updated organization 'ByteDance' -> 'Bytedance Ltd' on 123 enrollments, Updated organization 'ByteDance' -> 'Bytedance Ltd' on 533 archived enrollments, Deleted organization 'ByteDance' (id=58791), Added organization 'Captalys Companhia de Credito' (id=77308), Updated organization 'Gupy' -> 'Captalys Companhia de Credito' on 8 enrollments, Deleted organization 'Gupy' (id=34128), Updated organization 'CloudCheckr' -> 'CloudCheckr, Inc.' on 6 enrollments, Updated organization 'CloudCheckr' -> 'CloudCheckr, Inc.' on 11 archived enrollments, Deleted organization 'CloudCheckr' (id=47724), Updated organization name 'Code For Africa' -> 'Code for Africa' (id=51260), Updated organization 'Codewise.com' -> 'Codewise Sp. z o.o.' on 9 enrollments, Deleted organization 'Codewise.com' (id=31626), Updated organization 'AngelList' -> 'Cyanoptics' on 6 enrollments, Deleted organization 'AngelList' (id=46814), Updated organization 'Datto' -> 'Datto, Inc.' on 46 enrollments, Updated organization 'Datto' -> 'Datto, Inc.' on 6 archived enrollments, Deleted organization 'Datto' (id=29196), Updated organization 'Deloitte Canada' -> 'Deloitte Consulting LLP' on 12 enrollments, Updated organization 'Deloitte Canada' -> 'Deloitte Consulting LLP' on 6 archived enrollments, Deleted organization 'Deloitte Canada' (id=38895), Updated organization 'Deloitte Consulting' -> 'Deloitte Consulting LLP' on 7 enrollments, Updated organization 'Deloitte Consulting' -> 'Deloitte Consulting LLP' on 5 archived enrollments, Deleted organization 'Deloitte Consulting' (id=48484), Added organization 'DENX Software Engineering GmbH (DENX)' (id=77309), Updated organization 'DENX Computer Systems' -> 'DENX Software Engineering GmbH (DENX)' on 126 enrollments, Updated organization 'DENX Computer Systems' -> 'DENX Software Engineering GmbH (DENX)' on 465 archived enrollments, Deleted organization 'DENX Computer Systems' (id=26313), Added organization 'Digi Telecommunications Sdn BhdÂ ' (id=77310), Updated organization 'Digi' -> 'Digi Telecommunications Sdn BhdÂ ' on 12 enrollments, Updated organization 'Digi' -> 'Digi Telecommunications Sdn BhdÂ ' on 7 archived enrollments, Deleted organization 'Digi' (id=41811), Updated organization 'XebiaLabs' -> 'Digital.ai Software, Inc.' on 50 enrollments, Updated organization 'XebiaLabs' -> 'Digital.ai Software, Inc.' on 19 archived enrollments, Deleted organization 'XebiaLabs' (id=32648), Updated organization 'de Persgroep Nederland' -> 'DPG Media' on 8 enrollments, Updated organization 'de Persgroep Nederland' -> 'DPG Media' on 6 archived enrollments, Deleted organization 'de Persgroep Nederland' (id=38645), Updated organization 'EngineerBetter' -> 'EngineerBetter Ltd' on 39 enrollments, Updated organization 'EngineerBetter' -> 'EngineerBetter Ltd' on 109 archived enrollments, Deleted organization 'EngineerBetter' (id=59184), Updated organization 'EPITA:' -> 'EPITA' on 7 enrollments, Updated organization 'EPITA:' -> 'EPITA' on 16 archived enrollments, Deleted organization 'EPITA:' (id=45091), Updated organization 'Da Wanda' -> 'Etsy, Inc.' on 6 enrollments, Deleted organization 'Da Wanda' (id=42350), Added organization 'F6S Network Limited' (id=77311), Updated organization 'F6S' -> 'F6S Network Limited' on 23 enrollments, Deleted organization 'F6S' (id=59201), Updated organization '패스트캠퍼스' -> 'Fast campus' on 5 enrollments, Deleted organization '패스트캠퍼스' (id=32863), Updated organization 'Flipkart Online Services Pvt. Ltd.' -> 'Flipkart Internet Private Limited' on 9 enrollments, Deleted organization 'Flipkart Online Services Pvt. Ltd.' (id=59218), Added organization 'FrÃ¡vega SACIeI' (id=77312), Updated organization 'Frávega' -> 'FrÃ¡vega SACIeI' on 6 enrollments, Deleted organization 'Frávega' (id=44161), Added organization 'FundaciÃ³n Instituto Superior de EnseÃ±anza' (id=77313), Updated organization 'ISE' -> 'FundaciÃ³n Instituto Superior de EnseÃ±anza' on 5 enrollments, Deleted organization 'ISE' (id=44961), Added organization 'GESTIÃ“N TRIBUTARIA TERRITORIAL SA' (id=77314), Updated organization 'Gestión Tributaria Territorial' -> 'GESTIÃ“N TRIBUTARIA TERRITORIAL SA' on 6 enrollments, Updated organization 'Gestión Tributaria Territorial' -> 'GESTIÃ“N TRIBUTARIA TERRITORIAL SA' on 7 archived enrollments, Deleted organization 'Gestión Tributaria Territorial' (id=47703), Added organization 'Globo ComunicaÃ§Ã£o e ParticipaÃ§Ãµes S.A' (id=77315), Updated organization 'Rede Globo' -> 'Globo ComunicaÃ§Ã£o e ParticipaÃ§Ãµes S.A' on 6 enrollments, Deleted organization 'Rede Globo' (id=48068), Added organization 'Grupo OesÃ­a' (id=77316), Updated organization 'Oesia' -> 'Grupo OesÃ­a' on 15 enrollments, Updated organization 'Oesia' -> 'Grupo OesÃ­a' on 60 archived enrollments, Deleted organization 'Oesia' (id=42567), Added organization 'Healthcare of Ontario Pension Plan' (id=77317), Updated organization 'HOOPP' -> 'Healthcare of Ontario Pension Plan' on 11 enrollments, Deleted organization 'HOOPP' (id=44038), Updated organization 'Hewlett-Packard' -> 'Hewlett Packard Enterprise Development LP' on 15 enrollments, Updated organization 'Hewlett-Packard' -> 'Hewlett Packard Enterprise Development LP' on 25 archived enrollments, Deleted organization 'Hewlett-Packard' (id=49035), Added organization 'Honeylogic OÃœ' (id=77318), Updated organization 'HoneyLogic' -> 'Honeylogic OÃœ' on 4 enrollments, Updated organization 'HoneyLogic' -> 'Honeylogic OÃœ' on 46 archived enrollments, Deleted organization 'HoneyLogic' (id=50146), Added organization 'Horizon Robotics' (id=77319), Updated organization '地平线｜边缘人工智能芯片全球领导者' -> 'Horizon Robotics' on 9 enrollments, Updated organization '地平线｜边缘人工智能芯片全球领导者' -> 'Horizon Robotics' on 6 archived enrollments, Deleted organization '地平线｜边缘人工智能芯片全球领导者' (id=54626), Updated organization 'Huawei  Telecommunication-Guangdong' -> 'Huawei Technologies Co., Ltd' on 8 enrollments, Updated organization 'Huawei  Telecommunication-Guangdong' -> 'Huawei Technologies Co., Ltd' on 24 archived enrollments, Deleted organization 'Huawei  Telecommunication-Guangdong' (id=41260), Updated organization 'IIT Hyderabad' -> 'Indian Institute of Technology Hyderabad' on 5 enrollments, Updated organization 'IIT Hyderabad' -> 'Indian Institute of Technology Hyderabad' on 4 archived enrollments, Deleted organization 'IIT Hyderabad' (id=58216), Updated organization 'Geek Gazette' -> 'Indian Institute of Technology Roorkee' on 5 enrollments, Updated organization 'Geek Gazette' -> 'Indian Institute of Technology Roorkee' on 100 archived enrollments, Deleted organization 'Geek Gazette' (id=34626), Updated organization 'Infor Pty Ltd' -> 'Infor Global Solutions' on 20 enrollments, Updated organization 'Infor Pty Ltd' -> 'Infor Global Solutions' on 117 archived enrollments, Deleted organization 'Infor Pty Ltd' (id=59278), Added organization 'IngenieurbÃ¼ro Gietzen' (id=77320), Updated organization 'Ingenieurbüro Gietzen' -> 'IngenieurbÃ¼ro Gietzen' on 6 enrollments, Deleted organization 'Ingenieurbüro Gietzen' (id=46901), Added organization 'Institut national de la santÃ© et de la recherche mÃ©dicale (Inserm)' (id=77321), Updated organization 'INSERM' -> 'Institut national de la santÃ© et de la recherche mÃ©dicale (Inserm)' on 13 enrollments, Updated organization 'INSERM' -> 'Institut national de la santÃ© et de la recherche mÃ©dicale (Inserm)' on 4 archived enrollments, Deleted organization 'INSERM' (id=46603), Updated organization name 'Institute for Defense Analyses' -> 'Institute For Defense Analyses' (id=39881), Added organization 'Intelygenz Desarrollos InformÃ¡ticos SA' (id=77322), Updated organization 'intelygenz' -> 'Intelygenz Desarrollos InformÃ¡ticos SA' on 11 enrollments, Updated organization 'intelygenz' -> 'Intelygenz Desarrollos InformÃ¡ticos SA' on 251 archived enrollments, Deleted organization 'intelygenz' (id=49747), Updated organization 'Iowa State University of Science and Technology' -> 'Iowa State University of Science  and Technology' on 11 enrollments, Updated organization 'Iowa State University of Science and Technology' -> 'Iowa State University of Science  and Technology' on 42 archived enrollments, Deleted organization 'Iowa State University of Science and Technology' (id=47382), Added organization 'Ispirata' (id=77323), Updated organization 'Ispirata Srl' -> 'Ispirata' on 6 enrollments, Updated organization 'Ispirata Srl' -> 'Ispirata' on 4 archived enrollments, Deleted organization 'Ispirata Srl' (id=44007), Updated organization 'Yuso A Flit' -> 'Karhoo' on 5 enrollments, Deleted organization 'Yuso A Flit' (id=44764), Updated organization 'NAGRA' -> 'Kudelski S.A' on 6 enrollments, Updated organization 'NAGRA' -> 'Kudelski S.A' on 2 archived enrollments, Deleted organization 'NAGRA' (id=54390), Added organization 'KÃ©rozen interactive media' (id=77324), Updated organization 'Kérozen médias interactifs' -> 'KÃ©rozen interactive media' on 8 enrollments, Deleted organization 'Kérozen médias interactifs' (id=46928), Updated organization name 'Lifestreams' -> 'LifeStreams' (id=41793), Updated organization 'MC Research Labs' -> 'MasterControl Inc' on 6 enrollments, Deleted organization 'MC Research Labs' (id=47045), Updated organization 'The MathWorks INC' -> 'MathWorks, Inc.' on 43 enrollments, Updated organization 'The MathWorks INC' -> 'MathWorks, Inc.' on 70 archived enrollments, Deleted organization 'The MathWorks INC' (id=59639), Updated organization 'Medium Corp.' -> 'Medium' on 16 enrollments, Deleted organization 'Medium Corp.' (id=59383), Updated organization 'Merkle Group Inc.' -> 'Merkle Inc.' on 2 enrollments, Deleted organization 'Merkle Group Inc.' (id=71827), Updated organization 'Merkle|RKG' -> 'Merkle Inc.' on 5 enrollments, Updated organization 'Merkle|RKG' -> 'Merkle Inc.' on 1 archived enrollments, Deleted organization 'Merkle|RKG' (id=47065), Updated organization 'Net IQ' -> 'Micro Focus International plc' on 6 enrollments, Updated organization 'Net IQ' -> 'Micro Focus International plc' on 8 archived enrollments, Deleted organization 'Net IQ' (id=41764), Added organization 'MinistÃ¨re de lâ€™Ã©ducation nationale, de la jeunesse et des sports' (id=77325), Updated organization 'Ministère de l'éducation nationale' -> 'MinistÃ¨re de lâ€™Ã©ducation nationale, de la jeunesse et des sports' on 6 enrollments, Deleted organization 'Ministère de l'éducation nationale' (id=46737), Added organization 'monday.com Limited' (id=77326), Updated organization 'monday.com' -> 'monday.com Limited' on 6 enrollments, Deleted organization 'monday.com' (id=50242), Added organization 'MyScript' (id=77327), Updated organization 'MyScript®' -> 'MyScript' on 6 enrollments, Deleted organization 'MyScript®' (id=47136), Updated organization 'Nasqad Inc.' -> 'Nasdaq, Inc.' on 32 enrollments, Updated organization 'Nasqad Inc.' -> 'Nasdaq, Inc.' on 18 archived enrollments, Deleted organization 'Nasqad Inc.' (id=59421), Updated organization 'NBN Australia' -> 'NBN Co Limited' on 6 enrollments, Deleted organization 'NBN Australia' (id=55384), Added organization 'Object Partners, Inc.' (id=77328), Updated organization 'Object Partners' -> 'Object Partners, Inc.' on 8 enrollments, Deleted organization 'Object Partners' (id=49661), Added organization 'Odine Solutions Teknoloji Tic. ve San. A. Åž.' (id=77329), Updated organization 'Odine' -> 'Odine Solutions Teknoloji Tic. ve San. A. Åž.' on 5 enrollments, Deleted organization 'Odine' (id=51130), Added organization 'Oxyliom Solutions SÃ rl' (id=77330), Updated organization 'Oxyliom Solutions' -> 'Oxyliom Solutions SÃ rl' on 12 enrollments, Deleted organization 'Oxyliom Solutions' (id=43588), Updated organization 'PA Consulting Group' -> 'PA Consulting Holdings Limited' on 7 enrollments, Updated organization 'PA Consulting Group' -> 'PA Consulting Holdings Limited' on 2 archived enrollments, Deleted organization 'PA Consulting Group' (id=49168), Updated organization 'Seapine Software' -> 'Perforce Software Inc.' on 15 enrollments, Updated organization 'Seapine Software' -> 'Perforce Software Inc.' on 15 archived enrollments, Deleted organization 'Seapine Software' (id=45791), Updated organization 'PLVision' -> 'PL Vision' on 22 enrollments, Updated organization 'PLVision' -> 'PL Vision' on 29 archived enrollments, Deleted organization 'PLVision' (id=45131), Updated organization 'Points.com' -> 'Points International Ltd.' on 24 enrollments, Deleted organization 'Points.com' (id=28863), Added organization 'Probely â€“ Cybersecurity Solutions, Lda' (id=77331), Updated organization 'Probely' -> 'Probely â€“ Cybersecurity Solutions, Lda' on 12 enrollments, Deleted organization 'Probely' (id=46451), Updated organization 'Procter & Gamble' -> 'Proctor & Gamble Company' on 7 enrollments, Updated organization 'Procter & Gamble' -> 'Proctor & Gamble Company' on 1 archived enrollments, Deleted organization 'Procter & Gamble' (id=54586), Added organization 'PROMESS Montage- und PrÃ¼fsysteme GmbH' (id=77332), Updated organization 'PROMESS Montage' -> 'PROMESS Montage- und PrÃ¼fsysteme GmbH' on 6 enrollments, Deleted organization 'PROMESS Montage' (id=46353), Added organization 'Providence Health and Services' (id=77333), Updated organization 'Providence Health & Services' -> 'Providence Health and Services' on 13 enrollments, Deleted organization 'Providence Health & Services' (id=31483), Updated organization 'Qihoo 360' -> 'Qihoo 360 Technology Co. Ltd' on 21 enrollments, Deleted organization 'Qihoo 360' (id=54873), Updated organization 'Qihoo 360 Technology Co., Ltd.' -> 'Qihoo 360 Technology Co. Ltd' on 15 enrollments, Updated organization 'Qihoo 360 Technology Co., Ltd.' -> 'Qihoo 360 Technology Co. Ltd' on 51 archived enrollments, Deleted organization 'Qihoo 360 Technology Co., Ltd.' (id=26452), Updated organization 'Quental' -> 'Quental Technologies S.L.' on 8 enrollments, Deleted organization 'Quental' (id=51231), Updated organization 'Raytheon' -> 'Raytheon Technologies' on 48 enrollments, Updated organization 'Raytheon' -> 'Raytheon Technologies' on 120 archived enrollments, Deleted organization 'Raytheon' (id=26820), Added organization 'Rikagaku KenkyÅ«jo (Riken)' (id=77334), Updated organization 'Riken' -> 'Rikagaku KenkyÅ«jo (Riken)' on 14 enrollments, Deleted organization 'Riken' (id=31397), Updated organization 'Salesforce.com Inc.' -> 'Salesforce.com, inc.' on 7 enrollments, Updated organization 'Salesforce.com Inc.' -> 'Salesforce.com, inc.' on 14 archived enrollments, Deleted organization 'Salesforce.com Inc.' (id=59543), Added organization 'Sigorta Bilgi ve GÃ¶zetim Merkezi (SBM)' (id=77335), Updated organization 'Sigorta Bilgi ve Gözetim Merkezi' -> 'Sigorta Bilgi ve GÃ¶zetim Merkezi (SBM)' on 6 enrollments, Deleted organization 'Sigorta Bilgi ve Gözetim Merkezi' (id=46770), Added organization 'SingleStore, Inc.' (id=77336), Updated organization 'MemSQL Inc.' -> 'SingleStore, Inc.' on 5 enrollments, Deleted organization 'MemSQL Inc.' (id=59388), Updated organization 'MemSQL, Inc.' -> 'SingleStore, Inc.' on 5 enrollments, Deleted organization 'MemSQL, Inc.' (id=51993), Added organization 'SocialBase SoluÃ§Ãµes em Tecnologia SA' (id=77337), Updated organization 'SocialBase' -> 'SocialBase SoluÃ§Ãµes em Tecnologia SA' on 5 enrollments, Deleted organization 'SocialBase' (id=42843), Added organization 'Software Assurance Marketplace  (SWAMP)' (id=77338), Updated organization 'Snow B.V.' -> 'Sue B.V.' on 5 enrollments, Updated organization 'Snow B.V.' -> 'Sue B.V.' on 6 archived enrollments, Deleted organization 'Snow B.V.' (id=51510), Updated organization 'Synopsys Inc.' -> 'Synopsys, Inc' on 11 enrollments, Updated organization 'Synopsys Inc.' -> 'Synopsys, Inc' on 5 archived enrollments, Deleted organization 'Synopsys Inc.' (id=59612), Added organization 'TecnologÃ­a, Sistemas y Aplicaciones, S.L.' (id=77339), Updated organization 'Tecsisa' -> 'TecnologÃ­a, Sistemas y Aplicaciones, S.L.' on 22 enrollments, Updated organization 'Tecsisa' -> 'TecnologÃ­a, Sistemas y Aplicaciones, S.L.' on 60 archived enrollments, Deleted organization 'Tecsisa' (id=43706), Updated organization 'TELUS Digital' -> 'TELUS Communications Inc.' on 5 enrollments, Deleted organization 'TELUS Digital' (id=43275), Updated organization name 'teuto.net Netzdienste GmbH' -> 'Teuto.net Netzdienste GmbH' (id=33136), Updated organization 'Sina Com Technology' -> 'Tianjin Chengtay Technology Co., Ltd' on 62 enrollments, Updated organization 'Sina Com Technology' -> 'Tianjin Chengtay Technology Co., Ltd' on 51 archived enrollments, Deleted organization 'Sina Com Technology' (id=59569), Added organization 'Universitat Politècnica de Catalunya' (id=77340), Updated organization 'Polytechnic University of Catalonia' -> 'Universitat Politècnica de Catalunya' on 6 enrollments, Deleted organization 'Polytechnic University of Catalonia' (id=55013), Updated organization 'Universitat Politècnica de Catalunya' -> 'UPC (Universitat Politècnica de Catalunya)' on 6 enrollments, Deleted organization 'Universitat Politècnica de Catalunya' (id=77340), Updated organization 'oDesk' -> 'Upwork Global Inc.' on 9 enrollments, Updated organization 'oDesk' -> 'Upwork Global Inc.' on 66 archived enrollments, Deleted organization 'oDesk' (id=57988), Updated organization 'U.S. Department of Defense' -> 'US Department of Defense' on 5 enrollments, Updated organization 'U.S. Department of Defense' -> 'US Department of Defense' on 2 archived enrollments, Deleted organization 'U.S. Department of Defense' (id=32910), Updated organization 'Nynex' -> 'Verizon Corporate Services' on 6 enrollments, Deleted organization 'Nynex' (id=44327), Updated organization 'Oath Inc. dba Verizon Media' -> 'Verizon Media' on 44 enrollments, Updated organization 'Oath Inc. dba Verizon Media' -> 'Verizon Media' on 172 archived enrollments, Deleted organization 'Oath Inc. dba Verizon Media' (id=51629), Updated organization 'Vivo Communication' -> 'vivo Mobile Communication Co., Ltd.' on 7 enrollments, Deleted organization 'Vivo Communication' (id=44042), Updated organization 'vivo Mobile Communication Co., Ltd' -> 'vivo Mobile Communication Co., Ltd.' on 12 enrollments, Deleted organization 'vivo Mobile Communication Co., Ltd' (id=69083), Added organization 'wunschlÃ¶sung GmbH' (id=77341), Updated organization 'wunschlösung' -> 'wunschlÃ¶sung GmbH' on 11 enrollments, Deleted organization 'wunschlösung' (id=49427), Added organization 'WÃ¼rth Phoenix S.r.l.' (id=77342), Updated organization 'Würth Phoenix' -> 'WÃ¼rth Phoenix S.r.l.' on 6 enrollments, Updated organization 'Würth Phoenix' -> 'WÃ¼rth Phoenix S.r.l.' on 2 archived enrollments, Deleted organization 'Würth Phoenix' (id=42355), Updated organization 'ZTE' -> 'ZTE Corporation' on 1214 enrollments, Updated organization 'ZTE' -> 'ZTE Corporation' on 1431 archived enrollments, Deleted organization 'ZTE' (id=55382), Added organization 'Ã…hlÃ©ns AB' (id=77343), Updated organization 'Åhléns AB' -> 'Ã…hlÃ©ns AB' on 5 enrollments, Updated organization 'Åhléns AB' -> 'Ã…hlÃ©ns AB' on 2 archived enrollments, Deleted organization 'Åhléns AB' (id=47617), Organizations: added:44 renamed:5 deleted:107 skipped:0, Enrollments: conflicts:0 archive conflicts:0 updated:11924"}
```

Then next run results (one circular org):
```
{"text":"Added organization 'Universitat Politècnica de Catalunya' (id=77344), Deleted organization 'Universitat Politècnica de Catalunya' (id=77344), Organizations: added:1 renamed:0 deleted:1 skipped:0, Enrollments: conflicts:0 archive conflicts:0 updated:0"}
```

Sorry, gotta go - will be back in few hours.

**Comment by bumblebee89** [comment]
@lukaszgryglicki OH, i meant for it to be test not master. Thanks for taking care of this. I ll create PRs for test and prod with the fix for circular org

**Pull request merged**